ui/src/pages/plan/rules/RulesContainer.tsx

```diff
@@ -24,17 +25,6 @@ export const RulesContainer = ({ userid, onRefresh = () => {} }: { userid: strin
         onRefresh()
     }, [refetch, onRefresh])
 
-    const deleteHandler = useCallback((id: string) => {
-        axios.delete(`${baseUrl}/api/rules/${id}?userid=${userid}`)
-            .then(() => {
-                triggerRefresh();
-            })
-            .catch((e) => {
-                // TODO: toast an error
-                console.error('UHOH', e);
-            })
-    }, [triggerRefresh, userid]);
```

Comment:
Now needs to call close modal, so moved later in the file component so the dependency for the hook would be defined